org.hl7.fhir.utilities/src/main/java/org/hl7/fhir/utilities/npm/NpmPackageIndexBuilder.java

```diff
@@ -57,7 +57,7 @@ public void start(String filename) {
             "Content        nvarchar NULL,\r\n"+
             "PRIMARY KEY (FileName))\r\n");
 
-        psql = conn.prepareStatement("Insert into index (FileName, ResourceType, Id, Url, Version, Kind, Type, Supplements, Content) values (?, ?, ?, ?, ?, ?, ?, ?, ?)");
+        psql = conn.prepareStatement("Insert into ResourceList (FileName, ResourceType, Id, Url, Version, Kind, Type, Supplements, Content) values (?, ?, ?, ?, ?, ?, ?, ?, ?)");
       } catch (Exception e) {
         conn = null;

```

Comment:
This looks like it may leave a hanging connection when exceptions are caught. Don't know if it could cause hangups on trying to delete or rewrite.

Consider:

```java
if (conn != null) { conn.close(); }
```

Reply:
ok